core/storage/btree.rs

```diff
@@ -67,11 +74,16 @@ enum WriteState {
 }
 
 struct WriteInfo {
+    /// State of the write operation state machine.
     state: WriteState,
+    /// Pages allocated during the write operation due to balancing.
     new_pages: RefCell<Vec<PageRef>>,
+    /// Scratch space used during balancing.
     scratch_cells: RefCell<Vec<&'static [u8]>>,
+    /// Bookkeeping of the rightmost pointer so the PAGE_HEADER_OFFSET_RIGHTMOST_PTR can be updated.
     rightmost_pointer: RefCell<Option<u32>>,
-    page_copy: RefCell<Option<PageContent>>, // this holds the copy a of a page needed for buffer references
+    /// Copy of the current page needed for buffer references.
```

Comment:
I don't really know what this means tbh

Reply:
What do you mean?

Reply:
I guess I meant the comment would help more if there was a description of why a copy is needed in the context of balancing, but it's not a big deal